src/GongSolutions.WPF.DragDrop/DragDrop.cs

```diff
@@ -583,16 +603,14 @@ private static void DropTargetOnDragOver(object sender, DragEventArgs e, EventTy
 
             dropHandler.DragOver(dropInfo);
 
-            if (DragAdorner == null && dragInfo != null)
+            if (DragDropPreview is null && dragInfo is { })
```

Comment:
`is { }` is new to me, neat. I usually do `is object` to prevent `!= null`

Reply:
Both are new to me, but `is not null` would be really better. I used the "To object pattern"...

![image](https://user-images.githubusercontent.com/658431/97635842-5450bc00-1a38-11eb-9845-3e11a037a27a.png)